content/en/blog/2025/otel-js-sdk-2-0.md

```diff
@@ -0,0 +1,85 @@
+- The **minimum supported TypeScript version has been raised to 5.0.4**.
+- The **compilation target for transpiled TypeScript has been raised to ES2022**
+  (from ES2017).
+- The **public interface has changed**
+  - for notes on migrating to 2.x / 0.200.x see
+    [the upgrade guide](https://github.com/open-telemetry/opentelemetry-js/tree/main/doc/upgrade-to-2.x.md)
+
+### Details
+
+- [Node.js supported versions](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#-nodejs-supported-versions)
+- [TypeScript supported versions](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#-typescript-supported-versions)
+- [ES2022 compilation target](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#-es2022-compilation-target)
+- [Drop `window.OTEL_*` support in browsers](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#-drop-windowotel_-support-in-browsers)
+- [`@opentelemetry/resources` API changes](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#-opentelemetryresources-api-changes)
+- [`@opentelemetry/core` API changes](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#-opentelemetrycore-api-changes)
+- [Tracing SDK API changes](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#-tracing-sdk-api-changes)
+- [`@opentelemetry/sdk-metrics` API changes](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#-opentelemetrysdk-metrics-api-changes)
+- [`@opentelemetry/resources` changes for _implementors_ of Resource Detectors](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#-opentelemetryresources-changes-for-implementors-of-resource-detectors)
+- [Other changes](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#-other-changes)
+
+## Why was this done?
+
+Some of the details for why this 2.0 was done can be found
+[in this issue](https://github.com/open-telemetry/opentelemetry-js/issues/4083).
+We knew we would gain the greatest benefit by allowing breaking changes to
+improve things related to:
+
+- Optimization: removing classes and namespaces to allow better minification and
+  tree-shaking.
+- Better Tooling and Support: dropping old runtimes and tool versions to take
+  advantage of enhanced ESM support and simplify documentation.
+- Velocity: reducing code complexity and removing deprecated fields for faster
+  feature and maintenance work.
+
+This also begins our goal of releasing a new major release every year.
+
+## How can I get involved?
+
+- Try out the
+  [v2.0.0](https://github.com/open-telemetry/opentelemetry-js/releases/tag/v2.0.0)
+  and
+  [v0.200.0](https://github.com/open-telemetry/opentelemetry-js/releases/tag/experimental%2Fv0.200.0)
+  releases and provide feedback
```

Comment:
do you want to add something on how to provide feedback? I see people getting confused on how to do so. Maybe mention creation of issues, or reaching our slack/SIG. I know you added slack and sig right below, but wanted to make more clear those are the communications channels for feedback too